lib/smart_proxy_openscap/foreman_forwarder.rb

```diff
@@ -25,13 +25,14 @@ def upload_path(cname, policy_id, date)
       "/api/v2/compliance/arf_reports/#{cname}/#{policy_id}/#{date}"
     end
 
-    def send_request(path, body)
+    def send_request(path, body, timeout)
```

Comment:
this seems to be also used at https://github.com/theforeman/smart_proxy_openscap/blob/master/lib/smart_proxy_openscap/fetch_file.rb#L49 either make timeout optional or modify it everywhere